src/pubsub.ts

```diff
@@ -772,23 +771,30 @@ export class PubSub {
    * @see [Schemas: list API Documentation]{@link https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.schemas/list}
    * @see [More about async iterators]{@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of}
    *
+   * @param {google.pubsub.v1.SchemaView} [view] The type of schema objects
+   *   requested, which should be an enum value from {@link SchemaViews}. Defaults
+   *   to Full.
    * @param {object} [options] Request configuration options, outlined
    *   here: https://googleapis.github.io/gax-nodejs/interfaces/CallOptions.html.
-   * @returns {AsyncIterable<Schema>}
+   * @returns {AsyncIterable<ISchema>}
    *
    * @example
    * for await (const s of pubsub.listSchemas()) {
    *   const moreInfo = await s.get();
    * }
    */
-  async *listSchemas(options?: CallOptions): AsyncIterable<Schema> {
+  async *listSchemas(
+    view: google.pubsub.v1.SchemaView = SchemaViews.Full,
```

Comment:
Let's use BASIC as the default for ListSchemas.

FULL should be the default for GetSchemas, as this code already does.